rtb/verbali/interni/VI_2024-11-10_ver1.0.0.typ

```diff
@@ -0,0 +1,75 @@
+#import "/templates/template.typ": *
+
+#show: content=>verbale(
+  data: "10 novembre 2024", 
+  responsabile: "Riccardo Milan", 
+  redattori: ("Andrea Precoma",), 
+  titolo: "Terza riunione", 
+  verificatori: (
+    "Andrea Perozzo",
+    "Davide Picello",
+    "Klaudio Merja"
+  ),
+  riassunto: "Discussione principale riguardo alla struttura del backlog, alla suddivisione delle repository e al sistema di pubblicazione della documentazione. Miglioramento del Way of Working.",
+  versioni: (
+    "1.0.0", 
+    [10/11/2024], 
+    "Andrea Precoma", 
+    "Andrea Perozzo\nDavide Picello\nKlaudio Merja",
+    [- Stesura del verbale],
+  ),
+  content: content
+)
+
+#infoRiunione(
+  data: "10/11/2024", durata: "2 ore e 10 minuti", luogo: "Chiamata Discord", ora: "15:15", partecipantiEsterni: (),
+)
+
+= Ordine del giorno
+== Riunione esterna con l'azienda proponente
+In seguito alla vittoria della candidatura per il capitolato C4 abbiamo provveduto a pianificare una e-mail per lunedì 11 novembre per concordare un incontro con l'azienda proponente. Abbiamo chiesto disponibilità dopo il "Diario di Bordo" e la lezione rovesciata in modo da avere qualche chiarimento anche dal professor Vardanega e avere più materiale sul quale discutere.
+
+== Sito per la documentazione
+Dopo alcune proposte stilistiche riguardo al sito dove pubblicare la documentazione abbiamo deciso di aggiornarci alla prossima riunione. Oltre a sistemare l'aspetto del sito bisogna aggiungere la logica che permette di collegare i link del sito ai documenti in pdf. Abbiamo quindi creato la repository *sweatunipd.github.io* che conterrà i file Python che aggiorneranno i link del sito in modo che aprano i documenti desiderati.
+
+== Backlog
```

Comment:
Non mi convince una cosa: diciamo di usare le issue come backlog, per poi passare a dire che la board funge da backlog... unirei issue e board (cioè le stesse compongono la board) in un'unica frase